plugins/mcreator-localization/lang/texts.properties

```diff
@@ -1757,6 +1773,8 @@ elementgui.block.is_ladder=<html>Does block act like ladder?<br><small>Block mus
 elementgui.block.enchantments_bonus=Enchantments power bonus:
 elementgui.block.flammability=Block flammability:
 elementgui.block.push_reaction=Reaction to being pushed:
+elementgui.block.properties_general=General properties
+elementgui.block.properties_dropping=Dropping properties
```

Comment:
Please remove these texts that don't relate to any of the stuff in this PR, check if there are more like this

Reply:
Uh... why Intellij? 🤔 😂

Reply:
I checked and it seems they were the only ones.